packages/123done/oauth.js

```diff
@@ -48,6 +48,7 @@ function verifyIdToken(oauthConfig, token) {
 
 function setupOAuthFlow(req, action, options = {}, cb) {
   var params = {
+    access_type: 'offline',
```

Comment:
I added this so that 123done would receive refresh tokens and we could add a functional test to the content server to ensure this is fixed.